x-pack/plugins/apm/public/components/app/Main/UpdateBreadcrumbs.tsx

```diff
@@ -47,12 +42,16 @@ class UpdateBreadcrumbsComponent extends React.Component<Props> {
   }
 }
 
-const flatRoutes = flatten(
-  routes.map(route => (route.switchRoutes ? route.switchRoutes : route))
-);
-
-const UpdateBreadcrumbs = withBreadcrumbs(flatRoutes)(
-  UpdateBreadcrumbsComponent
+const UpdateBreadcrumbs = () => (
+  <ProvideBreadcrumbs
+    routes={routes}
+    render={({ breadcrumbs, location }) => (
+      <UpdateBreadcrumbsComponent
+        breadcrumbs={breadcrumbs}
+        location={location}
+      />
+    )}
+  />
 );
 
 export { UpdateBreadcrumbs };
```

Comment:
I think you told me, but what is the advantage of exporting like this versus `export function UpdateBreadcrumbs`?